### DISCUSSION TITLE
doctrine-pear.php license

### DISCUSSION THREAD

**Discussion by mipr99** [doctrine/doctrine2#8882] [Support Questions] [answered]
Hello,
I have a question regarding licensing. I know the Doctrine ORM is covered by the MIT License however the file bin/doctrine-pear.php mentions LGPL license. I would like to ask which license then refers to this file? I suppose it's MIT but I would like to be sure.

**Comment by beberlei** [comment, marked as answer]
Its MIT licensed, we changed from LGPL about 8 years ago or longer. This file header was probably forgotten

In addition our support for PEAR has ended, so this file will be removed at some point or already is iin newer branches.

**Reply by mipr99** [reply]
Thank you.
